README.md

```diff
@@ -0,0 +1,117 @@
+#  NoiteDev
+[![Build Status](https://travis-ci.org/condessalovelace/mavenquickstart.svg?branch=master)](https://travis-ci.org/condessalovelace/mavenquickstart) ![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=br.com%3Amavenquickstart&metric=alert_status)
+
+## Welcome in my repository 
+
+![prototype-003](./src/assets/prototype-003.png)
+
+## Description 
+
+This project illustrates an example of a simple application for storing portuguese to english words with pouchDB, a free and offline database for learning.
```

Comment:
```suggestion
This project illustrates an example of a simple application for storing portuguese to english words with pouchDB, a free and offline database for learning.
```
This project illustrates an example of a simple application for storing words in Portuguese and your respective English translate using pouchDB, for be a free and offline database for learning.